src/js/mobile.js

```diff
@@ -121,6 +121,7 @@
 			"./core/widget/core/Tabs",
 			"./profile/mobile/widget/GridView",
 			"./profile/mobile/expose",
+			"./core/widget/core/DatePicker",
```

Comment:
This file should be './profile/mobile/widget/DatePicker'

Reply:
ok. I will update and change file locate.